backend/src/graphql/resolver/TransactionResolver.ts

```diff
@@ -112,11 +114,41 @@ export class TransactionResolver {
     const self = new User(user)
     const transactions: Transaction[] = []
 
+    const transactionLinkRepository = getCustomRepository(TransactionLinkRepository)
+    const { sumHoldAvailableAmount, sumAmount } = await transactionLinkRepository.sumAmounts(
+      user.id,
+      now,
+    )
+
     // decay transaction
     if (!onlyCreations && currentPage === 1 && order === Order.DESC) {
       transactions.push(
         virtualDecayTransaction(lastTransaction.balance, lastTransaction.balanceDate, now, self),
       )
+      // open transaction link sum transaction
```

Comment:
Please rethink your naming conventions. Who would ever understand intuitively what `sumHoldAvailableAmount` stands for? Sorry for the hate! ❤️ 

```suggestion
      // virtual transaction for pending transaction-links sum 
```